src/controllers/accommodation.js

```diff
@@ -113,5 +113,21 @@ export default class accomodationFacility {
       return res.status(500).json({ status: 500, errorMessage: error });
     }
   }
+
+  static async mostTraveled(req, res) {
+    const traveledCentre = await models.Bookings.findAll({
```

Comment:
Why not using Booking from imports?

Reply:
I have switched to imported  you can check